pep-0678.rst

```diff
@@ -229,6 +245,49 @@ proposed ``__note__`` semantics, but this would be rarely and inconsistently
 applicable.
 
 
+Augment the ``raise`` statement
+-------------------------------
+One discussion proposed ``raise Exception() with "note contents"`` , but this
+does not address the original motivation of compatibility with ``ExceptionGroup`` .
+
+Furthermore, this would be an additional feature over adding the attribute and
+modifying the traceback-formatting code, and we do not expect ``__note__`` to be
+so frequently-used as to justify new syntax or even a new ``.with_note()`` method
+(analogous to ``.with_traceback()`` ).
+
+
+Allow any object, and cast to string for display
+------------------------------------------------
+We have not identified any scenario where libraries would want to do anything but either
+concatenate or replace notes, and so the additional complexity and interoperability
+challenges do not seem justified.
+
+Variations on this have also been proposed for forward-compatibility with a future
+structured API, but in the absence of any proposed use-case (see also "Non-goals")
+we strongly prefer to keep things simple - and note that implementing any nontrival
+future behavior would then break backwards-compatibility.
+
+
+Add a helper function ``contexlib.add_exc_note()``
+--------------------------------------------------
+We don't expect notes to be used frequently enough to justify this either, but
+provide the following extensible recipe::
```

Comment:
This is the second time you’re saying you rejected an idea because you don’t think notes will be used frequently enough to justify it. I don’t think this is the reason.

The context manager is an enhancement we could add any time in the future. I don’t think you’re really rejecting it, just not including it in the basic spec of the feature. Maybe you need a section for ‘possible future enhancements’.